lib/logger/log.js

```diff
@@ -0,0 +1,9 @@
+const winston = require('winston');
+let logger = new (winston.Logger)({
+    transports : [
+        new (winston.transports.Console)(),
+        new (winston.transports.File)({filename:'./logger/info.log'})
```

Comment:
You need not use File transport for local development. Console transport works fine. You need to disable file transport based on the presence (or absence) of a flag